data/quickstarts/oneapi-aikit-notebook.yaml

```diff
@@ -0,0 +1,95 @@
+apiVersion: console.openshift.io/v1
+kind: ConsoleQuickStart
+metadata:
+  name: oneapi-aikit-notebook
```

Comment:
The aikit application def specifies a non-existent quick start `create-aikit-notebook`, if this is to be the default quick start (I assume it would be) then either the name of this needs to be `create-aikit-notebook` or we need to update https://github.com/red-hat-data-services/odh-dashboard/blob/master/data/applications/aikit.yaml#L16

Reply:
thanks @jeff-phillips-18 
I fixed this is the latest commit.